commit-message.YAML-tmLanguage

```diff
@@ -54,18 +54,26 @@ patterns:
     - name: comment.line.number-sign.git-commit
       match: (^#)\s*(modified:.*)$\n?
       captures:
-        '0': {name: markup.changed.git-commit}
         '1': {name: punctuation.definition.comment.git-commit}
+        '2': {name: markup.changed.git-commit}
+    - name: comment.line.number-sign.git-commit
+      match: (^#)\s*(renamed:)\s*(.*)\s*(->)\s*(.*)$\n?
```

Comment:
All in all, everything’s good, thank you.

But this line [match for renamed] looks weird:
![2016-06-03_18-45-03](https://cloud.githubusercontent.com/assets/1975298/15779043/e9f6336a-29bb-11e6-9c82-1d57e85c16ec.png)

My first thought was `(^#)\s*(renamed:)\s*(.*(?= )) (->)\s*(.*)$` : 
![2016-06-03_18-46-55](https://cloud.githubusercontent.com/assets/1975298/15779064/09fc37c2-29bc-11e6-8a1e-53ad86f3e7c7.png)

But then other lines probably should have separated backgrounds too. Hm…

What your thoughts are?